README.md

```diff
@@ -52,6 +52,8 @@ This will install the latest release of PCT to `~/.puppetlabs/pct`.
 
 ![install_pct](docs/_resources/install_and_export_path.gif)
 
+> :warning: If you do not use the install script and are extracting the archive yourself, be sure to use the fully qualified path to `~/.puppetlabs/pct` when you set your PATH environment variable.
```

Comment:
will windows users who use this approach know that `~/` means the user home directory?

Reply:
Good point. I'd say most but not all. I'll modify this